Dockerfile

```diff
@@ -159,6 +160,10 @@ RUN fname="stargz-snapshotter-${STARGZ_SNAPSHOTTER_VERSION}-${TARGETOS:-linux}-$
   rm -f "${fname}" /out/bin/stargz-store && \
   mv stargz-snapshotter.service /out/lib/systemd/system/stargz-snapshotter.service && \
   echo "- Stargz Snapshotter: ${STARGZ_SNAPSHOTTER_VERSION}" >> /out/share/doc/nerdctl-full/README.md
+ARG SOCI_SNAPSHOTTER_VERSION
```

Comment:
You are adding soci to "build-full" stage, we need it to be in test stage. After line 272.